ci-scripts/firebase_publish.py

```diff
@@ -56,7 +56,7 @@ def main():
         print(f"Decoding build_metadata_env JSON failed: {build_metadata_env}")
     build_info = {"metadata": build_metadata}
 
-    builds = ["AGW", "FEG", "ORC8R", "CWAG", "NMS"]
+    builds = ["FEG", "ORC8R", "CWAG", "NMS"]
     for build in builds:
         artifact_env = os.environ[f"{build}_ARTIFACTS"]
```

Comment:
one could instead have a "if environment has `f"{build}_ARTIFACTS"` in env" guard?

Reply:
follow up created: #15159